client/src/main/java/com/sap/mobile/services/client/push/ApnsNotification.java

```diff
@@ -439,7 +442,21 @@ public Builder mutableContent(Boolean mutableContent) {
 					this.localizedAlertTitleArguments, this.alertSubtitle, this.localizedAlertSubtitleKey,
 					this.localizedAlertSubtitleArguments, this.launchImageFileName, this.showActionButton,
 					this.actionButtonLabel, this.localizedActionButtonKey, this.urlArguments, this.threadId,
-					mutableContent);
+					mutableContent, this.interruptionLevel);
+		}
+
+		/**
+		 * The interruption-level is optional and defines the importance and delivery
+		 * timing of a notification
+		 */
+		public Builder interruptionLevel(InterruptionLevel interruptionLevel) {
+			return new Builder(this.expiration, this.category, this.contentAvailable, this.pushType, this.customValues,
+					this.sound, this.customSound, this.topic, this.alertBody, this.localizedAlertKey,
+					this.localizedAlertArguments, this.alertTitle, this.localizedAlertTitleKey,
+					this.localizedAlertTitleArguments, this.alertSubtitle, this.localizedAlertSubtitleKey,
+					this.localizedAlertSubtitleArguments, this.launchImageFileName, this.showActionButton,
+					this.actionButtonLabel, this.localizedActionButtonKey, this.urlArguments, this.threadId,
+					this.mutableContent, this.interruptionLevel);
```

Comment:
```suggestion
					this.mutableContent, interruptionLevel);
```

`interruptionLevel` parameter must be passed to the new builder instance.

Reply:
Thanks!
See my added test cases